resources/js/mosaic.pattern.js

```diff
@@ -224,15 +225,17 @@ export default Base.extend({
             }
         });
 
-        // on enabling: hide toolbar, add mosaic class, disable body toolbar classes
-        $("#edit-bar").hide();
-        var $body = $("body");
-        $body.addClass("mosaic-enabled");
-        $body[0].className.split(" ").forEach(function (className) {
-            if (className.indexOf("plone-toolbar") !== -1) {
-                $body.removeClass(className);
-            }
-        });
+        if (self.options.disable_edit_bar) {
+            // on enabling: hide toolbar, add mosaic class, disable body toolbar classes
+            $("#edit-bar").hide();
+            var $body = $("body");
+            $body.addClass("mosaic-enabled");
+            $body[0].className.split(" ").forEach(function (className) {
+                if (className.indexOf("plone-toolbar") !== -1) {
+                    $body.removeClass(className);
+                }
```

Comment:
```suggestion
            document.getElementById("edit-bar").style.display = "none";
            document.body.classList.add("mosaic-enabled");
            document.body.classList.remove("plone-toolbar");
```

1. I started to remove jQuery in every change I make in mosaic ... can you test my suggestions above?
2. the body class "mosaic-enabled" is now only added when the toolbar is hidden. As looking into the scss files, there are some rules that apply to `body.mosaic-enabled` ... does this work for you?